text/0497-glue-l2-construct.md

```diff
@@ -0,0 +1,434 @@
+# RFC - Glue CDK L2 Construct
+
+## L2 Construct for AWS Glue Connections, Jobs, and Workflows
+
+* **Original Author(s):** @natalie-white-aws, @mjanardhan @parag-shah-aws
+* **Tracking Issue:**
+* **API Bar Raiser:** @TheRealAmazonKendra
+[Link to RFC Issue](https://github.com/aws/aws-cdk-rfcs/issues/497)
+
+## Working Backwards - README
```

Comment:
My comment on this first section of working backwards is that we're not really working backwards from a README. A lot of the material here is (necessary) context, but would not be information found in a README.

Reply:
We'll rename the section.